docs/rules/file-not-included.md

```diff
@@ -0,0 +1,28 @@
+---
+title: file-not-included
+---
+
+## Why?
+
+Although generally uncommon, sometimes including a change log in the root deltas change log is forgotten.
+
+The `file-not-included` rule will fail if any file in the specified target directories were not included.
+The specified directory **is** searched recursively. The check is also only performed against files which have
+the same file extension as the root deltas change log. For example, if you are using `xml` change logs, a `*.json`
+file would be ignored.
+
+## Options
+
+`values` - (array of strings) list of relative paths to include
```

Comment:
```suggestion
`values` - (array of strings) list of paths relative to the working directory Liquibase is being run from that should be checked for not-included files
```

Reply:
What do we do if no `values` provided? From the code looks like we don't support that, which I think is fine. Should we throw a meaningful exception in that case?

I think it would also be worth highlighting why in the docs - i.e. too much chance of false positives with other bits of XML/JSON/YAML, potential usages of `includeAll` etc

Reply:
Addressed slightly tweaked your suggestion in the end